### DISCUSSION TITLE
Using the WoT Thing Model

### DISCUSSION THREAD

**Discussion by fix-all** [eclipse-ditto/ditto#1908] [General]
hi,
   I have created a Thing instance using the Thing Model, which includes 'tm:extends' and 'tm:submodel' in the links. Now, I need to validate the Thing instance using the definition. However, using '/api/2/things/{thingId}/definition' only provides the URL of the definition, not the complete content. Is there any related API that I can use to obtain the complete content of the definition?

**Comment by thjaeckle** [comment]
Sure, as documented using content negotiation: https://eclipse.dev/ditto/basic-wot-integration.html#thing-description-generation

**Reply by fix-all** [reply]
You are right, thanks
